include/nabto/nabto_device_experimental.h

```diff
@@ -55,6 +55,31 @@ nabto_device_set_private_key_secp256r1(NabtoDevice* device, const uint8_t* key,
 NABTO_DEVICE_DECL_PREFIX NabtoDeviceError NABTO_DEVICE_API
 nabto_device_disable_remote_access(NabtoDevice* device);
 
+/**
+ * Add a key-value pair to the metadata of a TCP tunnel service.
+ *
```

Comment:
what happens if a key already exists is the old one overwritten?

Reply:
If a key already exists then ideally it should be overwritten, however nn_string_map's insert function does not overwrite by default. 1dfd669 updates the documentation and the code to make it overwrite the old key-value pair.